templates/default/customer/customerinfobox.html

```diff
@@ -50,7 +50,7 @@
 						{block name="customerinfobox-leftcolumn"}
 							<TR>
 								<TD WIDTH="1%">
-									<IMG src="img/{$_CSTATUSES[$customerinfo.status].img}" alt="">
+									<i class="lms-ui-icon-user-state"></i>
```

Comment:
tutaj jest regres trzeba inne stany klienta tez obikonkować

Reply:
Mogę akurat tej ikony nie zmieniać póki co.

Reply:
lms-ui-icon-customer-state-{$_CSTATUSES[$customerinfo.status]}

Tak powinno działać jeśli zrobisz odpowiednie klasy w pliku less&CSS.

Reply:
lms-ui-icon-customer-state-{$_CSTATUSES[$customerinfo.status].alias}
Tak powinno działać jeśli zrobisz odpowiednie klasy w pliku less&CSS.

Reply:
> lms-ui-icon-customer-state-{$_CSTATUSES[$customerinfo.status].alias}
> Tak powinno działać jeśli zrobisz odpowiednie klasy w pliku less&CSS.

Dorobię to dzisiaj.

Reply:
Dodane właśnie w 13ea5de. Powinno działać.